File: src/main/java/com/graqr/threshr/model/redsky/product/Image.java
package com.graqr.threshr.model.redsky.product;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record Image(
        String url,
        @JsonProperty("alt_text")
        String altText,
        Long height,
        Long width,
        @JsonProperty("aspect_ratio")
        String aspectRatio) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/Image.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class Image(
    val url: String,
    @param:JsonProperty("alt_text") val altText: String,
    val height: Long,
    val width: Long,
    @param:JsonProperty("aspect_ratio") val aspectRatio: String
) {}